hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -22,6 +22,67 @@
       milestone: V1
```

Comment:
typo at line 14 (I can't comment at that line), nedded -> needed

Reply:
corrected